Documentation/dev/libvirt-howto.md

```diff
@@ -1,33 +1,31 @@
-# Libvirt howto
+# Libvirt HOWTO
 
 Tectonic has limited support for installing a Libvirt cluster. This is useful especially
 for operator development.
 
-## HOW TO:
-### 1. One-time setup
+## 1. One-time setup
 It's expected that you will create and destroy clusters often in the course of development. These steps only need to be run once (or once per RHCOS update).
 
-#### 1.1 Pick a name and ip range
+### 1.1 Pick a name and ip range
 In this example, we'll set the baseDomain to `tt.testing`, the name to `test1` and the ipRange to `192.168.124.0/24`
 
-#### 1.2 Clone the repo
+### 1.2 Clone the repo
 ```sh
 git clone https://github.com/openshift/installer.git
 cd installer
 ```
 
-#### 1.3 (Optional) Download and prepare the operating system image
+### 1.3 (Optional) Download and prepare the operating system image
 
 *By default, the installer will download the latest RHCOS image every time it is invoked. This may be problematic for users who create a large number of clusters or who have limited network bandwidth. The installer allows a local image to be used instead.*
 
 Download the latest RHCOS image (you will need access to the Red Hat internal build systems):
 
 ```sh
-wget http://aos-ostree.rhev-ci-vms.eng.rdu2.redhat.com/rhcos/images/cloud/latest/rhcos-qemu.qcow2.gz
-gunzip rhcos-qemu.qcow2.gz
+curl http://aos-ostree.rhev-ci-vms.eng.rdu2.redhat.com/rhcos/images/cloud/latest/rhcos-qemu.qcow2.gz | gunzip > rhcos-qemu.qcow2
 ```
```

Comment:
There was a step in @sjenning first screencast instructing how to set enforcing=0 in the qcow image with virt-edit, if that's still necessary we should add that here.

Reply:
I was about to test it again with that image but the URL doesn't work anymore. Has it changed?

Reply:
works for me.. are you still on the vpn?

Reply:
I don't remember being on the VPN for getting that image. Is there a public mirror?

Reply:
> Is there a public mirror?

[No][1], although I don't know what the reasoning is.

[1]: https://github.com/openshift/installer/issues/281#issuecomment-422942411

Reply:
> There was a step in @sjenning first screencast instructing how to set enforcing=0 in the qcow image with virt-edit, if that's still necessary we should add that here.

It is not necessary anymore